tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/TinkerGraphFeatureTest.java

```diff
@@ -33,7 +33,7 @@
         tags = "not @RemoteOnly and not @GraphComputerOnly and not @AllowNullPropertyValues",
         glue = { "org.apache.tinkerpop.gremlin.features" },
         objectFactory = TinkerGraphFeatureTest.TinkerGraphGuiceFactory.class,
-        features = { "classpath:/org/apache/tinkerpop/gremlin/test/features" },
+        features = { "classpath:/org/apache/tinkerpop/gremlin/test/features/map/DateDiff.feature" },
```

Comment:
you can control tests being executed with a system variable so that you don't need to touch this file and see it accidentally committed: `-Dcucumber.filter.tags="@StepDateDiff"`